feat(content): Korath Emergency Ships (Rai'alorej & Ikatila'ej)

ravenshining opened the pull request:
----------------------
**Content (Artwork)**

Note:  This is a re-PRing of #7174 which was closed so that the merge conflicts could be fixed and the attendant fleets and events could be added more quickly and easily.

## Summary
This PR adds the final two large ships from #4987 to the Korath Exiles: The hospital ship Rai'alorej, and the factory ship Ikatila'ej, both extensively remodelled by Saugia.

The Rai'alorej-class Refuge is hospital ship that chases after battles to rescue injured Korath.  It is spec'd for speed and defense, not offense, carries two ambulances, and is the only civilian Korath ship seen outside the core.

![rai'alorej](https://user-images.githubusercontent.com/93169396/187014336-705a0486-f11e-438c-965c-5ed0ff3d9340.png)

The extremely rare Ikatila'ej-class Refabricator is a flying factory that uses microbots to repair and produce critical supplies for broken-down stations and world-ships.

![ikatila'ej](https://user-images.githubusercontent.com/93169396/187014327-ced063dc-974c-470d-b23c-b281486aa7b5.png)

When not assisting damaged ships and stations, it mines asteroids, drawing them in to its fearsome maw to be torn apart and processed on-board.  Starship captains are advised to avoid flying into this maw themselves.

![grinder](https://user-images.githubusercontent.com/93169396/187043950-c9fd5dcc-b228-4de6-a314-d010a5a93ea9.png)

As always, you can find out more about this and other Korath content on our discord server, Swizzle 6
https://discord.gg/hNGWFA3fjB

## Artwork Checklist
 - [x] I updated the copyright attributions, or decline to claim copyright of any assets produced or modified
 - [x] I created a PR to the [endless-sky-assets repo](https://github.com/EndlessSkyCommunity/endless-sky-assets) with the necessary image, blend, and texture assets: https://github.com/endless-sky/endless-sky-high-dpi/pull/290
 - [x] I created a PR to the [endless-sky-high-dpi repo](https://github.com/endless-sky/endless-sky-high-dpi) with the `@2x` versions of these art assets: https://github.com/EndlessSkyCommunity/endless-sky-assets/pull/79

Saugia commented:
To clarify, I closed the PR because not only was never given a single clear answer as to how we want to go about this PR's implementation (after being harassed and argued with for similar PR's), I was censored for not "being calm" at your assumptiuous discretion. I was in no rush to PR this at all. 

That's why I opened https://github.com/endless-sky/endless-sky/pull/7260, so that I may continue to pursue all of the effort I put into Korath artwork without constantly being argued with and treated like I'm upset when I'm not. While I do appreciate you kept the original PR's work intact, I would strongly advise to contact me next time, or at the very least attribute me in the description PR as was done in Stage ii. 

I will kindly ask only once that any future PR's that contain my work be done so through Zitchas as I can speak with him just fine.

Saugia commented:
So it's not as much work, here are the other two PR's restored.

https://github.com/endless-sky/endless-sky-high-dpi/pull/290
https://github.com/EndlessSkyCommunity/endless-sky-assets/pull/79

ravenshining commented:
Of course I deny these allegations harrassment, arguement, censorship, or refusal to answer.

I simply had yet to make up my mind on the question, and asked to be left alone until I was able to answer.

> That's why I opened #7260 ... I would strongly advise to contact me next time, or at the very least attribute me in the PR's description as was done in Stage ii.

I was not aware of #7260 .  I gathered from the way that you left the server, closed your PRs, and deleted your branches, that you did not wish to be bothered.

I did mention you in the description, just without the @ as I didn't want to make an attempt to contact you if you wished to be left alone.  By the same token, however, I would appreciate it if others were to @ me when drafting PRs that contain or affect my work.

Saugia commented:
>Of course I deny these allegations harrassment, arguement, censorship, or refusal to answer.

>I simply had yet to make up my mind on the question, and asked to be left alone until I was able to answer.

You harassed me for both the Tubfalet and 'nra'ret PR's for not being introduced the way you would like even after being told multiple times that was the plan between me and Zitchas, and started to do it with the last one as well. 

I did not say you didn't refused to answer, but you never gave a clear one as to when you want to do this PR and I had *calmly* tried asking you in different ways to help you understand. Your response was to censor me and delete all my messages. This sort of behavior is not befitting of a faction lead whatsoever and I have reason to believe you're no longer the fit despite defending you to my core up until now. You can't just treat everything that happens like a personal situation.

ravenshining commented:
GitHub is not the place to be holding an arguement.  If you have issues with me or my Discord moderating style, kindly bring them up in DM and I will be happy do discuss them.

Saugia commented:
While I agree, just because you choose not to discuss things in ESC for personal reasons, I now have reason not to do the same with you. Please respect that if you want me to respect your side.

warp-core commented on data/korath/korath weapons.txt:
```diff
@@ -1011,6 +1011,35 @@ effect "firelight ring"
+		"inaccuracy" 10
+		"velocity" 10
+		"lifetime" 2
+		"reload" 6
+		"firing energy" 5
+		"blast radius" 100
+		"shield damage" 48
+		"corrosion damage" 3
+		"minable damage" 200
+		"firing hull" -100
```
50 energy per second for 1000 hull repair out of 112 tonnes is better than every other hull repair outfit we have.

Hurleveur replied:
I think you forgot to add the keyword that prevents the player from looting it?

ravenshining replied:
Okay, it's bigger and unplunderable now.  Also not matched up at all, I figured it's got some empty space and isn't all weapon.

UnorderedSigh replied:
The Refabricator is a big microbot generator/control ship. In other words, the entire ship is the hull repair part of a systems core. It should have a crazy repair rate. Using a weapon is a creative way to do it. The original Refabricator just had built-in hull repair.

warp-core replied:
The negative firing hull should really either have a lower magnitude or the firing energy should be much higher, like at least 20, probably 30x what it currently is.

Amazinite replied:
This looks to me like a case of the game doesn't support the mechanics needed to do what we want to do (#2004), so we shouldn't try to force it.

warp-core commented on data/remnant/remnant 2 cognizance.txt:
```diff
@@ -193,6 +195,8 @@ mission "Remnant: Cognizance 4"
 	to complete
 		has "remnant: cognizance: found korath fleet"
 	on complete
+		event "korath rescue: aescolanus medium"
+		event "korath rescue: aescolanus medium end" 18
```
We didn't necessarily destroy the ships here.

ravenshining replied:
Ah, I see.  Reading up on the wiki, I take it I need to add an `action` under `branch destroyed`
Or have a silent mission trigger if that branch's condition is set

warp-core replied:
You can use the same action that is triggered if the ships are all destroyed.

warp-core commented on data/remnant/remnant events.txt:
```diff
@@ -129,6 +157,34 @@ event "korath hunting"
 
+event "korath rescue: nenia small"
+	system Nenia
+		add fleet "Korath Rescue" 2000
```
This is a fairly low period for a system so far from home.

ravenshining replied:
This isn't a random patrol, it's a response to a specific event in a known system.

warp-core reviewed (commented):
Could you give the hardpoint names double digit numbers? The same will need to be done on the high DPI PR.

roadrunner56 commented on data/korath/korath ships.txt:
```diff
@@ -387,6 +387,171 @@ ship "'olofez"
+	explode "huge explosion" 50
+	"final explode" "final explosion large"
+	description "Although fierce in battle and highly adaptable, Korath do not easily recover from injury. Without the aid of a surgeon or more equipment than fits into a standard escape pod, a stranded, injured Korath faces poor odds of survival. Therefore, Korath who leave their home territory have placed great importance on the construction of hospital ships that can quickly locate, recover, and  ...
+	description "	The Rai'alorej-class Refuge, named for a pioneer in the field of microbot-assisted surgery, is thoroughly appointed with lifeform scanners, locator beacon receivers, ambulances, and search-and-rescue teams. Once onboard, crash victims are greeted with the finest hospital facilities, microbot treatments, and stores of medicine the Exiles have available. Even incubators and nurseries ...
```
```suggestion
	description "	The Rai'alorej-class Refuge, named for a pioneer in the field of microbot-assisted surgery, is thoroughly appointed with lifeform scanners, locator beacon receivers, ambulances, and search-and-rescue teams. Once onboard, crash victims are greeted with the finest hospital facilities, microbot treatments, and stores of medicine the Exiles have at hand. Even incubators and nurseries are available to treat the young who grow up aboard ship."
```

tibetiroka commented on data/korath/korath ships.txt:
```diff
@@ -387,6 +387,171 @@ ship "'olofez"
+	explode "huge explosion" 50
+	"final explode" "final explosion large"
+	description "Although fierce in battle and highly adaptable, Korath do not easily recover from injury. Without the aid of a surgeon or more equipment than fits into a standard escape pod, a stranded, injured Korath faces poor odds of survival. Therefore, Korath who leave their home territory have placed great importance on the construction of hospital ships that can quickly locate, recover, and  ...
+	description "	The Rai'alorej-class Refuge, named for a pioneer in the field of microbot-assisted surgery, is thoroughly appointed with lifeform scanners, locator beacon receivers, ambulances, and search-and-rescue teams. Once onboard, crash victims are greeted with the finest hospital facilities, microbot treatments, and stores of medicine the Exiles have at hand. Even incubators and nurseries a ...
```
> who grow up aboard ship

This is probably fine, but I'd ask some other grammar people if we need to change it to `ships` or `a ship` or something.

Zitchas replied:
I think "aboard ship" is something of a saying in and of itself, similar to "onboard."

roadrunner56 replied:
I personally think it's fine.

Zireael07 replied:
From what I can tell by googling, it should be "aboard *a/the* ship", OR just "aboard" similar to "onboard"

ravenshining replied:
Aboard ship is an extremely common phrase.

UnorderedSigh replied:
The Elenchus has Hai that can communicate with Korath, but the player does not have a communicator yet. That gives you an easy way to allow the player to communicate with Korath: visit Elenchus and have the Korath tweak the translator. Once you can speak with the Korath though, that causes trouble in the Wanderers campaign. You have to be careful when and how to allow that.

ravenshining replied:
I don't remember if the conversation took place here or on Discord, but, I was given to understand that the Korath on Elenchus would not be speaking a Korath language, they'd have assimilated into the society and speak as everyone else does.

In any case, that would only give you one Korath language, I intend for there to be several which partially but not completely intelligible to one another, and then only by native speakers or linguistic experts, not by First Last.

UnorderedSigh replied:
Last I checked, the Korath at Elenchus speak something that is not Hai or Human Standard, and the Hai have to translate. They haven't assimilated in that sense, and so the story doesn't match what you've said. If it is to be as you say, someone needs to update the story.

MasterOfGrey replied:
> Last I checked, the Korath at Elenchus speak something that is not Hai or Human Standard, and the Hai have to translate. They haven't assimilated in that sense, and so the story doesn't match what you've said. If it is to be as you say, someone needs to update the story.

You checked wrong, the Korath speak (poorly) a language in common with the colony. What they speak amongst themselves is their own business, but would have suffered a number of generations of linguistic drift from its source either way.
They are not suitable to provide an in for a translator - they'd probably need one themselves to communicate with their own brethren.

tibetiroka reviewed (approved):
Grammar looks OK.

UnorderedSigh reviewed (approved):
This PR is okay how it is, but here are some problems I see:

1. The Rai'alorej has a small enough minimum crew that it could be captured with enough patience. Hypothetically, that could allow capturing of some of the larger ships. Not really a problem though, since it has no jump drive.
2. The Rai'alorej is slow for an emergency response vehicle. It's pretty fast for a gigantic monstrosity though.
3. In my tests, the Ikatila'ej didn't fare so well in combat.
4. The Rai'alorej still looks kindof flat. I think it's improved over the original though.
5. The medical emergency symbols on the back of the Rai'alorej are concealed by the tubes, which defeats the purpose of putting them there. (Moving the tubes would make it look more flat though.)

UnorderedSigh commented:
I'm glad you kept the rejuvenation dome forest. It shows the Korath still want to be around living things, even if they're not so good at keeping anything alive. I guess hospital Korath would be better than others at gardening.

ravenshining commented:
>     1. The Rai'alorej has a small enough minimum crew that it could be captured with enough patience. Hypothetically, that could allow capturing of some of the larger ships. Not really a problem though, since it has no jump drive.

It was intended to be a much smaller ship than the original Refuge, and shouldn't pose a capture ladder problem if we did the stats right.  Which we might not have.  It definitely *should* have a jump drive so we probably haven't.

>     5. The medical emergency symbols on the back of the Rai'alorej are concealed by the tubes, which defeats the purpose of putting them there. (Moving the tubes would make it look more flat though.)

I had no idea that's what those were, and presumably neither did Saugia.  I just assumed those were more machinery.

GefullteTaubenbrust2 reviewed (changes requested):
Art review time at long last. One thing that bothers me with the ships is the subsurf scatter. I know these have subsurf in the most unlikely of places, but it‘s most apparent with the tubes. It‘s not the end of the world because it doesn‘t show that much at the small scales, still it does bother me.

For the Rai-Alrorej, I feel like the greebles are a bit much. I know it‘s part of its lore, but basically the entire surface is covered with huge greebles, many just for greebles sake. It‘s also a little more messy than I feel like is appropriate, mainly with the tubes at the back, just flying about. Unless it‘s meant to be a meta social commentary on the state of the healthcare system, I don‘t feel like that‘s too appropriate for a hospital ship on the battlefield.

Ikatila-Ej looks quite good though. I like the shape, and it‘s in just about the right ballpark of messiness. I would infact be more lenient towards this thing being more messy than it is given how it‘s a factory where the Rai-Alrorej is a hospital ship, but on its own it looks great to me.

UnorderedSigh commented:
> For the Rai-Alrorej, I feel like the greebles are a bit much.

The abundance of greebles hides the fact that the ship is extremely flat. Otherwise it looks like you're fighting a baking pan. It's easier than redoing the whole ship. A consistent pattern of less messy greebles may work better, without having to redo the whole ship.

> I had no idea that's what those were, and presumably neither did Saugia. I just assumed those were more machinery.

NV put the yellow circles in to show that the Korath were identifying the ships as medical vessels, like we do on Earth with ambulances in war zones, usually with red crosses. The Korath never had Christianity (presumably) so a red cross wouldn't make sense. Hence, the strange symbol that confuses everyone. If you want to un-confuse them, you could mention it in the description.

> It was intended to be a much smaller ship than the original Refuge, and shouldn't pose a capture ladder problem if we did the stats right. Which we might not have. It definitely should have a jump drive so we probably haven't.

The Refuge was just below the Refabricator in the capture ladder, so if you haven't tweaked the stats, it's probably still capturable. I built the capture ladder for NV. You could get a Refabricator, but nothing larger. It took a lot of patience tho.

> tubes at the back, just flying about

Most of the Korath ships have tubes at the back near the engines. Making them super-messy is consistent with the rest of the ship being super-messy. Like @GefullteTaubenbrust2 said though, maybe a hospital ship should look sanitary, even if it is Korath.

ravenshining commented:
> The abundance of greebles hides the fact that the ship is extremely flat. Otherwise it looks like you're fighting a baking pan. It's easier than redoing the whole ship. A consistent pattern of less messy greebles may work better, without having to redo the whole ship.

I don't think it would be terribly difficult to give it a *little* depth, but that's probably a decision for @Saugia .  I'm certainly capable (although I should be focusing elsewhere) but I don't want to mess with his remodel without his go-ahead.

> Hence, the strange symbol that confuses everyone. If you want to un-confuse them, you could mention it in the description.

Or in either a Korath or Remnant mission.  Neat idea, I'll keep it in mind.

> The Refuge was just below the Refabricator in the capture ladder, so if you haven't tweaked the stats, it's probably still capturable. I built the capture ladder for NV. You could get a Refabricator, but nothing larger. It took a lot of patience tho.

Yeah, we've been re-doing the capture ladder, particularly since I added three more KWS variants.  And we should be massively tweaking the stats of the Ikatila'ej as it is now actually *smaller* than a KWS and is seen regularly throughout the Ember Wastes.

Currently the top rung of the ladder should be the Lor'nag Ik 590 (AKA KWS-D, transport).  There are plans to make one or two unique Kasichara A'awoj (AKA KWS-F, habitat) capturable.  The Rai'alorej would fit somewhere *below* those ships.  Whether we allow a ship as large and exotic as the Ikatila'ej to be capturable is a discussion for another time, for this PR, it is not.

> Most of the Korath ships have tubes at the back near the engines. Making them super-messy is consistent with the rest of the ship being super-messy. Like @GefullteTaubenbrust2 said though, maybe a hospital ship should look sanitary, even if it is Korath.

I agree with both of you :-)  The tubes fit the theme, but they are arranged a bit chaotically here, I think they ought to conform to the hull in a smooth pattern rather than float above it as if loose.

Saugia commented on data/korath/korath ships.txt:
```diff
@@ -387,6 +387,171 @@ ship "'olofez"
+		"Steering (Planetary Class)"
+		"Scram Drive"
+
+	"reverse engine" -82 -193.5
```
```suggestion
	"reverse engine" -79 -186
```

Saugia commented on data/korath/korath ships.txt:
```diff
@@ -387,6 +387,171 @@ ship "'olofez"
+
+	"reverse engine" -82 -193.5
+		zoom 0.8
+	"reverse engine" 82 -193.5
```
```suggestion
	"reverse engine" 79 -186
```

Saugia commented on data/korath/korath ships.txt:
```diff
@@ -387,6 +387,171 @@ ship "'olofez"
+		zoom 0.8
+	"reverse engine" 82 -193.5
+		zoom 0.8
+	engine -99 181.5
```
```suggestion
	engine -101 196
```

Saugia commented on data/korath/korath ships.txt:
```diff
@@ -387,6 +387,171 @@ ship "'olofez"
+		zoom 0.8
+	engine -99 181.5
+		zoom 0.9
+	engine 99 181.5
```
```suggestion
	engine 101 196
```

Saugia commented on data/korath/korath ships.txt:
```diff
@@ -387,6 +387,171 @@ ship "'olofez"
+		zoom 0.9
+	engine -60 167 0.8
+	engine 60 167 0.8
+	engine -29.5 195.5
```
```suggestion
	engine -29 212
```

Saugia commented on data/korath/korath ships.txt:
```diff
@@ -387,6 +387,171 @@ ship "'olofez"
+	engine -60 167 0.8
+	engine 60 167 0.8
+	engine -29.5 195.5
+	engine 29.5 195.5
```
```suggestion
	engine 29 212
```

Saugia commented on data/korath/korath ships.txt:
```diff
@@ -387,6 +387,171 @@ ship "'olofez"
+		zoom 0.9
+	engine 99 181.5
+		zoom 0.9
+	engine -60 167 0.8
```
```suggestion
	engine -60 190 0.8
```

Saugia commented on data/korath/korath ships.txt:
```diff
@@ -387,6 +387,171 @@ ship "'olofez"
+	engine 99 181.5
+		zoom 0.9
+	engine -60 167 0.8
+	engine 60 167 0.8
```
```suggestion
	engine 60 190 0.8
```

Saugia commented on data/korath/korath ships.txt:
```diff
@@ -387,6 +387,171 @@ ship "'olofez"
+	engine 60 167 0.8
+	engine -29.5 195.5
+	engine 29.5 195.5
+	gun -81.5 41.5 "Firelight Missile Bank"
```
```suggestion
	gun -81.5 58 "Firelight Missile Bank"
```

Saugia commented on data/korath/korath ships.txt:
```diff
@@ -387,6 +387,171 @@ ship "'olofez"
+	engine 29.5 195.5
+	gun -81.5 41.5 "Firelight Missile Bank"
+		angle 270
+	turret -88.5 -150 "Banisher Grav-Turret"
```
```suggestion
	turret -89 -134 "Banisher Grav-Turret"
```

Saugia commented on data/korath/korath ships.txt:
```diff
@@ -387,6 +387,171 @@ ship "'olofez"
+	gun -81.5 41.5 "Firelight Missile Bank"
+		angle 270
+	turret -88.5 -150 "Banisher Grav-Turret"
+	turret 88.5 -150 "Warder Anti-Missile"
```
```suggestion
	turret 89 -134 "Warder Anti-Missile"
```

Saugia commented on data/korath/korath ships.txt:
```diff
@@ -387,6 +387,171 @@ ship "'olofez"
+		angle 270
+	turret -88.5 -150 "Banisher Grav-Turret"
+	turret 88.5 -150 "Warder Anti-Missile"
+	turret -110.5 -142 "Warder Anti-Missile"
```
```suggestion
	turret -110 -126 "Warder Anti-Missile"
```

Saugia commented on data/korath/korath ships.txt:
```diff
@@ -387,6 +387,171 @@ ship "'olofez"
+	turret -88.5 -150 "Banisher Grav-Turret"
+	turret 88.5 -150 "Warder Anti-Missile"
+	turret -110.5 -142 "Warder Anti-Missile"
+	turret 110.5 -142 "Banisher Grav-Turret"
```
```suggestion
	turret 110 -126 "Banisher Grav-Turret"
```

Saugia commented on data/korath/korath ships.txt:
```diff
@@ -387,6 +387,171 @@ ship "'olofez"
+	turret 88.5 -150 "Warder Anti-Missile"
+	turret -110.5 -142 "Warder Anti-Missile"
+	turret 110.5 -142 "Banisher Grav-Turret"
+	turret 49.5 -53.5 "Langrage Hyper-Heaver"
```
```suggestion
	turret 36.5 -55.5 "Langrage Hyper-Heaver"
```

Saugia commented on data/korath/korath ships.txt:
```diff
@@ -387,6 +387,171 @@ ship "'olofez"
+	turret -110.5 -142 "Warder Anti-Missile"
+	turret 110.5 -142 "Banisher Grav-Turret"
+	turret 49.5 -53.5 "Langrage Hyper-Heaver"
+	turret -64 28 "Warder Anti-Missile"
```
```suggestion
	turret -63.5 44 "Warder Anti-Missile"
```

Saugia commented on data/korath/korath ships.txt:
```diff
@@ -387,6 +387,171 @@ ship "'olofez"
+	turret 110.5 -142 "Banisher Grav-Turret"
+	turret 49.5 -53.5 "Langrage Hyper-Heaver"
+	turret -64 28 "Warder Anti-Missile"
+	turret 64 28 "Banisher Grav-Turret"
```
```suggestion
	turret 63.5 44 "Banisher Grav-Turret"
```

Saugia commented on data/korath/korath ships.txt:
```diff
@@ -387,6 +387,171 @@ ship "'olofez"
+	turret 49.5 -53.5 "Langrage Hyper-Heaver"
+	turret -64 28 "Warder Anti-Missile"
+	turret 64 28 "Banisher Grav-Turret"
+	turret -64 50 "Banisher Grav-Turret"
```
```suggestion
	turret -63.5 66.5 "Banisher Grav-Turret"
```

Saugia commented on data/korath/korath ships.txt:
```diff
@@ -387,6 +387,171 @@ ship "'olofez"
+	turret -64 28 "Warder Anti-Missile"
+	turret 64 28 "Banisher Grav-Turret"
+	turret -64 50 "Banisher Grav-Turret"
+	turret 64 50 "Warder Anti-Missile"
```
```suggestion
	turret 63.5 66.5 "Warder Anti-Missile"
```

Saugia commented on data/korath/korath ships.txt:
```diff
@@ -387,6 +387,171 @@ ship "'olofez"
+	turret 64 28 "Banisher Grav-Turret"
+	turret -64 50 "Banisher Grav-Turret"
+	turret 64 50 "Warder Anti-Missile"
+	bay "Fighter" -119.5 -179
```
```suggestion
	bay "Fighter" -115 -152
```

Saugia commented on data/korath/korath ships.txt:
```diff
@@ -387,6 +387,171 @@ ship "'olofez"
+	turret 64 50 "Warder Anti-Missile"
+	bay "Fighter" -119.5 -179
+		angle 30
+	bay "Fighter" 119.5 -179
```
```suggestion
	bay "Fighter" 115 -152
```

Saugia commented:
Not sure why that submitted twice, thanks Github.

Zitchas commented:
> > Hence, the strange symbol that confuses everyone. If you want to un-confuse them, you could mention it in the description.
> 
> Or in either a Korath or Remnant mission. Neat idea, I'll keep it in mind.

I actually have a Remnant mission in development wherein the player is asked to go out and repair one of these ships. This would be an excellent opportunity for the briefer to inform the player about these markings and their significance. Along with mentioning to the player that the Remnant would not be thrilled with the player destroying/capturing said ships. This probably won't be PR'd for a month or so due to RW problems, but it is coming.

ravenshining commented:
That is a fantastic idea!  And a good motivation to keep players from attacking civilian ships.

Saugia commented:
@ravenshining I've gotten the go-ahead from Gef in DMs, once you swap the PR images with the ones I gave you he will be ready to approve.

ravenshining commented:
Right I'll do it later tonight.  As mentioned on discord I've a lot on my plate atm.

I'll double check the stats and fix the loadout while I'm at it.

GefullteTaubenbrust2 reviewed (approved):
Yeah I'd say it looks good.

warp-core commented:
Has anyone tested what it takes to capture either of these ships?

Hurleveur commented:
The Ikatila'ej looks unobtainable, except if you capture this first ship...
Which does not seem that hard to do.. I'll test it tomorrow but my 618 should be able to do it confortably

Hecter94 commented:
To paraphrase a conversation that happened elsewhere, to keep everyone involved in the loop:
At present, the presentation of these two ships seems to lack either a gameplay justification or a lore justification for existing. 

To ensure that we aren't adding things simply for the sake of adding them, I'd like one of the following questions answered:
1. Why do these ships need to be unique models, separate from the other production ships? What about the way Korath design their ships requires that these ships be unique?
2. What do these ships add gameplay-wise to the player? What mechanics do they have that support their roles in a way that other Korath ships do not? 

If we can get a good answer to one of those two questions, I think we'll have a good starting point for determining what the lore justification is for these ships existing as they do, or what the mechanical/gameplay justification is.

warp-core commented on data/korath/korath ships.txt:
```diff
@@ -387,6 +387,172 @@ ship "'olofez"
+		"heat dissipation" .29
+		"fuel capacity" 800
+		"cargo space" 845
+		"outfit space" 1829
+		"weapon capacity" 680
+		"engine capacity" 290
+		"spinal mount" 1
```
```suggestion
		"outfit space" 1153
		"weapon capacity" 680
		"engine capacity" 290
		"energy generation" 60
		"heat generation" 95
		"energy capacity" 15600
		"shield generation" 9
		"shield energy" 13.2
		"hull repair rate" 1.2
		"hull energy" 1.2
		"hull heat" 1.8
		"spinal mount" 1
```

Zitchas replied:
Ug. Not a fan. Why are we stopping it all the outfits in order to build all the same stuff into the hull?!

warp-core replied:
This ship has a ridiculous amount of outfit space (even by the standards of some other recent ships).
The microbot factory is also supposed to be a significant part of the ship, but as it is, it is essentially a battleship that happens to have a really close range gun on the front.
My alternative suggestion is that all the extra weapons be removed and it just be the factory on the front and the outfits to support that (and a systems core), but I suspect that would not be appreciated.

Why do we not want built in stats? I'm taking off the outfit space those outfits used to occupy, while it still has a similar effective outfit space, it is at least not customisable anymore.

Zitchas replied:
We are fine with the Bactrians being sort customizable. And w are equally fine with luxury quarters being used to host boarding canon fodder. Stuff being used for stuff other than what the original designers intended is part of the miracle and curse of highly modular ships. I don't see a reason to start locking down ships to prevent them from being useful to the player as desirable now.

warp-core commented on data/korath/korath ships.txt:
```diff
@@ -387,6 +387,172 @@ ship "'olofez"
+		"Grab-Strike Turret" 4
+		"Warder Anti-Missile" 5
+
+		"Triple Plasma Core"
+		"Double Plasma Core"
+		"Systems Core (Large)"
+		"Large Heat Shunt" 2
+		"Small Heat Shunt" 2
```
```suggestion
```

Zitchas replied:
This is the wrong direction. We want less stuff built-in, not more.

Hecter94 replied:
> This is the wrong direction. We want less stuff built in, not more.

This isn't quite true. 
Right now, we have some ships that have outfits built in, which aren't serving a purpose by being built-in. They aren't furthering lore by being built-in; they aren't providing a meaningful mechanical difference; they're built-in for convenience. 

Those are the kind of stats that we want to migrate out of ships and into outfits. 

However, in this case, this ship is entirely built for and designed around a single purpose. Therefore, having built-in stats like this is both lore appropriate and helps to differentiate this ship mechanically. 

A blanket statement of "built-in stats bad, don't have them" is remarkably limiting from a creative standpoint. We should make good use of all the available tools, not arbitrarily deciding that certain things are "bad" and should be avoided.

roadrunner56 replied:
A good example is the built-in cloak that Remnant ships use. It make sense that they're built in given the Remnant's history and culture; stealth is important to them. Making their cloak removable reduces their uniqueness and unbalances Remnant outfits. I'm not a stats guy so I don't know if built in outfits are a good idea in this case, but to make the blanket statement that built in outfits are bad, period, seems to be detrimental to the game.

Zitchas replied:
I agree, but I am also parroting Amazinite and others who have repeatedly told me that we should avoid building stuff into the hull if there is absolutely any way to accomplish the same thing using regular outfits.

ravenshining replied:
I'm not opposed to having stats built into the hull when it's something like a fighter that needs to squeeze out every tonne, or a Remnant ship whose cloaking is not something you could just outfit.

The idea of taking an existing outfit that the ship reasonably would have and building *that* into the hull, that is where it becomes a bad idea.

Why *wouldn't* a massive factory ship be able to have the space to swap out its reactors when one breaks down, or even lend one to someone who needs it?  This ship isn't starving for volume like a fighter, and the outfits that warp core is removing are not unique abilities.

One thing this is telling me, however, is that while I've been a bit back and forth as to whether or not this ship should be capturable, the fear that it might be stolen and used as a broken battleship tells me that it probably should not.  It's got enough microbots on board that it should be able to liquidate a KWS full of troops attempting to attack anyway, so the solution may just be to mark this uncapturable like the Korsmanath A'awoj.   Genocidal players may attempt to steal the outfits, perhaps even the grinder, but not the ship itself.

As much fun as it would be to mount a dragonflame on this and go on a rampage, such a ridiculous arrangement probably belongs to the realm of plugins.

warp-core commented on data/korath/korath weapons.txt:
```diff
@@ -1011,6 +1011,36 @@ effect "firelight ring"
+		"velocity" 10
+		"lifetime" 2
+		"reload" 6
+		"firing energy" 5
```
```suggestion
		"firing energy" 30
```

EjoThims commented:
1 is easy.

Unless I missed a major lore shift somewhere, all the various WS derivatives are made from sectioning up and modifying/adding onto the biggest WS in the early days of the Exiles adapting to station and ship life. What few are made "new" are, like the Palavret and Rano'erek, mostly made in sections and then only assembled in space.

Along with the Haven, these two are specialized hulls made wholesale in the few space docks that can manage. 'nra'rets are also assembled wholesale this way, as noted in their description.

These two utility ships should start even more rare than the Havens (especially in wild sightings instead of spawning from missions) and only grow more frequent as parts of fleets as milestones are reached in lore (through mission progression) that more are being produced. At the same time that they are made more common, WS derivative that fill the same roles should become far less so. Not only because they will not longer be made "new" as these are made more often, but because, after a certain point, each of these made means a competing WS derivative can be repurposed as pieces for other WS derivatives still in us.

The full progression of that wouldn't be appropriate to include in this PR (even were it yet sorted), though. It would be far better to include them alongside the events that contribute to the shift in production, and in separate, new PRs for any that would correlate to existing missions/events.

The gameplay impact is the demonstration of this lore even to characters cut off from a direct explanation of it, as well as the change in Korath presence, both hostile and non, as stories around the Korath are progressed even if their own is not.

If stats are a problem, then this may be an appropriate time to also adjust some of the WSes competing for these roles to ensure these ships actually provide upgrades.

ravenshining commented:
> At present, the presentation of these two ships seems to lack either a gameplay justification or a lore justification for existing.

All right.  It's late, I'm tired, I'm running low on power, and I'm extremely offended that you think there's no lore justification here.  So I'm basically just going to paste what I said on discord earlier:

These are distinct ships with distinct roles and lore to go with, not just yet another ship that looks identical to all the other ships, a problem that a number of other factions suffer from far worse than the Korath.

These ships provide functions that are not easily borne out by stats.  We'll mitigate that somewhat with the QoL outifits later, but there isn't a stat for "factory area" or "medical biodome"

The KWS is much smaller than, and doesn't have the manufacturing capabilities of, the Ikatila'ej.  And even if you built a KWS model that had the accommodations the Rai'alorej has for caring for to he wounded, you wouldn't want your enemy mistaking your hospital ship for a battleship.

And as Zitchas mentioned, he has plans for missions that highlight and rely on this aspect, which, as again already stated, also serve a gameplay function of incentivising the player to not anger the civilian faction.

It's also emblematic of the diversity of the Korath Exiles, to have a variety of visually distinct ship designs.  Although they've been forced together for many generations, they come from radically different cultures, and highly value their individuality and freedom of expression.  They are the pluralism to Efreti syncretism.

Hurleveur reviewed (commented):
Just some suggestions, feel free to follow them or not
I haven't suggested changes for the "Ikatila'ej" yet as the stats dont seem to be final (its got 285 free outfit space currently) but do feel free to hit me when that should be the case.

These ships do not benefit that much from having a reverser, other than being able to resist force pushing them back (which is why I kept a smaller one) they dont really use them to move around since they're so massive.

I also added rifles for the required crew due to turrets, and a bit of microbot stations, and capture defence for the big gun to balance the whole thing out and make them harder to capture.

Zitchas commented on data/korath/korath weapons.txt:
```diff
@@ -1011,6 +1011,36 @@ effect "firelight ring"
+		"minable damage" 200
+		"firing hull" -100
+		"piercing" 0.1
+		"safe"
```
This seems like a good change, especially in the future when we can specify that it isn't an atrocity for the Korath, but is for others.

Hurleveur replied:
oh right I always forget but I didnt really mean it should have 200 capture defence exactly, mostly an idea

ravenshining replied:
I think that's a great idea, but `atrocity` is a bit much.  I'll put `illegal` for now.  Maybe for the Korsmanath A'awoj in another PR though...

Saugia reviewed (changes requested):
Per discussions in discord:

Knock down turret amount to 5, change weapons to be 2 Heavers, 2 Warders, a central Banisher, and keeping the Firelight.
Adjusting hardpoints and space for this.
Adjusting internals by downgrading to a Double Plasma, upgrading to a LSC, and adding a Sys Core Med for defense/lore reasons. 
Buffing Microbot Station quantity for capturing concerns, fits lore.
Get rid of any extra empty space.

Hecter94 commented:
> These are distinct ships with distinct roles and lore to go with

What is the lore of these ships? 
This PR doesn't showcase that, or include anything beyond a description of the two ships and what their roles are. Nor is there anything player-facing that would give them any additional information on the lore of these ships.
If there is distinct lore that goes with these ships, or if you have plans to include such things in the future, that should be in the PR.

> These ships provide functions that are not easily borne out by stats. We'll mitigate that somewhat with the QoL outifits later, but there isn't a stat for "factory area" or "medical biodome"

What are these QoL outfits? There's no mention of this in the PR. 
What are they going to be adding to these ships? And why are we adding these ships now without those outfits if they're important for giving these ships the functions they should provide?
They should be included in this PR if they are part of the justification for merging this PR. Even if it's just a description for what you plan to add in the future, and why it's important to merge these ships now even though these outfits aren't ready yet.

> even if you built a KWS model that had the accommodations the Rai'alorej has for caring for the wounded, you wouldn't want your enemy mistaking your hospital ship for a battleship.

This seems like a weak justification. Surely there's a better method for ensuring your hospital ship isn't mistaken for a battleship. 
Do the other factions that the Exiles are going to be encountering know that this specific ship is a hospital ship and to avoid firing on it? If so, how? Is that information communicated some other way? 
If it has been communicated some other way, why couldn't one of their existing ships communicate that same information?

quyykk commented:
> Do the other factions that the Exiles are going to be encountering know that this specific ship is a hospital ship and to avoid firing on it?

We can do this by not given any weapons to the ship (except antimissile). Which also makes sense imo. Why would a hospital ship have weapons?

Hurleveur commented:
It has weapons to ward off those that attack it (*the player cough cough*)
I like how it is now more than before tbh for the hospital ship, and the other ship brings the aspect of what do they do with ships they cant salvage.

Zitchas commented:
The main faction that will be meeting the hospital ship is the Remnant, and it does know what they are and treat them as such. Which is to say, they will have missions informing the player that these did are not too be attacked, and will get upset if the player dues so. They will also repair these ships when necessary.

Hecter94 commented:
> The main faction that will be meeting the hospital ship is the Remnant, and it does know what they are and treat them as such. 

So are we saying that the Exiles built these ships specifically for the Remnant? If so, that still raises the question of:
"How was this information communicated to the Remnant, and why did it need to be a unique ship model in order to communicate that information?"

Considering the capability of the scanning technology the Remnant have shown so far, it seems remarkably unnecessary to have a separate ship model just to avoid them mistaking it for a battleship.

Hurleveur commented:
May as well make it obviously not a battleship instead of having doubts. It also has more bunks and is more agile. It should not be able to be a battleship ever, so that even in the chaos of battle it does not get shot at.

Also, the player needs to see it.

Hecter94 reviewed (approved):
Spoke with Raven and Saugia. This PR is fine as it is, my concerns will be addressed in future PRs

Amazinite commented on data/korath/korath weapons.txt:
```diff
@@ -1011,6 +1011,36 @@ effect "firelight ring"
+	"weapon capacity" -112
+	"required crew" 20
+	"gun ports" -1
+	"spinal mount" -1
```
I think this should lock from both directions. That is, while this has -1 and the ship has 1 spinal mount, there should be some other attribute where this has 1 and the ship has -1.

ravenshining replied:
It's intentional for it to be removable

Zitchas replied:
I agree with Ravenshining: I don't see a need for this to be locked in.

ravenshining replied:
There are no opportunities to cap this ship and take it to an outfitter at present, anyway.  Between that and it being `unplunderable` it's already locked to the ship.

In the future, however, if we ever get an implementation of #2004 , plus a facility to make hardpoint sprites appear *under* a ship, and some artistic revision, we could remove that tag and see this mounted on the other spinal ships as we originally intended.

Saugia commented on data/korath/korath ships.txt:
```diff
@@ -387,6 +387,172 @@ ship "'olofez"
+		"heat dissipation" .29
+		"fuel capacity" 800
+		"cargo space" 845
+		"outfit space" 1829
```
```suggestion
		"outfit space" 1314
```

Saugia commented on data/korath/korath ships.txt:
```diff
@@ -387,6 +387,172 @@ ship "'olofez"
+		"fuel capacity" 800
+		"cargo space" 845
+		"outfit space" 1829
+		"weapon capacity" 680
```
```suggestion
		"weapon capacity" 440
```

Saugia commented on data/korath/korath ships.txt:
```diff
@@ -387,6 +387,172 @@ ship "'olofez"
+		"cargo space" 845
+		"outfit space" 1829
+		"weapon capacity" 680
+		"engine capacity" 290
```
```suggestion
		"engine capacity" 259
```

Saugia commented on data/korath/korath ships.txt:
```diff
@@ -387,6 +387,172 @@ ship "'olofez"
+			"hit force" 20000
+	outfits
+		"Microbot Factory"
+		"Shunt-Strike Turret" 5
```
```suggestion
		"Shunt-Strike Turret" 3
```

Saugia commented on data/korath/korath ships.txt:
```diff
@@ -387,6 +387,172 @@ ship "'olofez"
+	outfits
+		"Microbot Factory"
+		"Shunt-Strike Turret" 5
+		"Grab-Strike Turret" 4
```
```suggestion
		"Grab-Strike Turret" 2
```

Saugia commented on data/korath/korath ships.txt:
```diff
@@ -387,6 +387,172 @@ ship "'olofez"
+		"Microbot Factory"
+		"Shunt-Strike Turret" 5
+		"Grab-Strike Turret" 4
+		"Warder Anti-Missile" 5
```
```suggestion
		"Warder Anti-Missile" 3
```

Saugia commented on data/korath/korath ships.txt:
```diff
@@ -387,6 +387,172 @@ ship "'olofez"
+		"Double Plasma Core"
+		"Systems Core (Large)"
+		"Large Heat Shunt" 2
+		"Small Heat Shunt" 2
```
```suggestion
```

Saugia commented on data/korath/korath ships.txt:
```diff
@@ -387,6 +387,172 @@ ship "'olofez"
+		"Large Heat Shunt" 2
+		"Small Heat Shunt" 2
+		"Korath Repeater Rifle"  957
+		"Microbot Defense Station" 20
```
```suggestion
		"Microbot Defense Station" 26
```

Saugia commented on data/korath/korath ships.txt:
```diff
@@ -387,6 +387,172 @@ ship "'olofez"
+		"Reverser (Lunar Class)"
+		"Steering (Stellar Class)"
+		"Thruster (Stellar Class)"
+		"Hyperdrive"
```
```suggestion
		"Scram Drive"
```

Saugia commented on data/korath/korath ships.txt:
```diff
@@ -387,6 +387,172 @@ ship "'olofez"
+	engine 27 249.5
+		zoom 0.8
+	gun 0 -190 "Microbot Factory"
+	turret -90.5 -176 "Grab-Strike Turret"
```
```suggestion
```

Saugia commented on data/korath/korath ships.txt:
```diff
@@ -387,6 +387,172 @@ ship "'olofez"
+		zoom 0.8
+	gun 0 -190 "Microbot Factory"
+	turret -90.5 -176 "Grab-Strike Turret"
+	turret 90.5 -176 "Grab-Strike Turret"
```
```suggestion
```

Saugia commented on data/korath/korath ships.txt:
```diff
@@ -387,6 +387,172 @@ ship "'olofez"
+	gun 0 -190 "Microbot Factory"
+	turret -90.5 -176 "Grab-Strike Turret"
+	turret 90.5 -176 "Grab-Strike Turret"
+	turret -95 -156 "Grab-Strike Turret"
```
```suggestion
```

Saugia commented on data/korath/korath ships.txt:
```diff
@@ -387,6 +387,172 @@ ship "'olofez"
+	turret -90.5 -176 "Grab-Strike Turret"
+	turret 90.5 -176 "Grab-Strike Turret"
+	turret -95 -156 "Grab-Strike Turret"
+	turret 95 -156 "Grab-Strike Turret"
```
```suggestion
```

Saugia commented on data/korath/korath ships.txt:
```diff
@@ -387,6 +387,172 @@ ship "'olofez"
+	turret 90.5 -176 "Grab-Strike Turret"
+	turret -95 -156 "Grab-Strike Turret"
+	turret 95 -156 "Grab-Strike Turret"
+	turret -51.5 -98.5 "Shunt-Strike Turret"
```
```suggestion
```

Saugia commented on data/korath/korath ships.txt:
```diff
@@ -387,6 +387,172 @@ ship "'olofez"
+	turret -95 -156 "Grab-Strike Turret"
+	turret 95 -156 "Grab-Strike Turret"
+	turret -51.5 -98.5 "Shunt-Strike Turret"
+	turret 51.5 -98.5 "Warder Anti-Missile"
```
```suggestion
```

Saugia commented on data/korath/korath ships.txt:
```diff
@@ -387,6 +387,172 @@ ship "'olofez"
+	turret 95 -156 "Grab-Strike Turret"
+	turret -51.5 -98.5 "Shunt-Strike Turret"
+	turret 51.5 -98.5 "Warder Anti-Missile"
+	turret 0 -94.5 "Shunt-Strike Turret"
```
```suggestion
	turret 89 -168 "Grab-Strike Turret"
```

Saugia commented on data/korath/korath ships.txt:
```diff
@@ -387,6 +387,172 @@ ship "'olofez"
+	turret -51.5 -98.5 "Shunt-Strike Turret"
+	turret 51.5 -98.5 "Warder Anti-Missile"
+	turret 0 -94.5 "Shunt-Strike Turret"
+	turret 0 14 "Shunt-Strike Turret"
```
```suggestion
	turret -89 -168 "Grab-Strike Turret"
```

Saugia commented on data/korath/korath ships.txt:
```diff
@@ -387,6 +387,172 @@ ship "'olofez"
+	turret 51.5 -98.5 "Warder Anti-Missile"
+	turret 0 -94.5 "Shunt-Strike Turret"
+	turret 0 14 "Shunt-Strike Turret"
+	turret -145.5 83 "Shunt-Strike Turret"
```
```suggestion
	turret -56 96 "Shunt-Strike Turret"
```

Saugia commented on data/korath/korath ships.txt:
```diff
@@ -387,6 +387,172 @@ ship "'olofez"
+	turret 0 -94.5 "Shunt-Strike Turret"
+	turret 0 14 "Shunt-Strike Turret"
+	turret -145.5 83 "Shunt-Strike Turret"
+	turret 145.5 83 "Warder Anti-Missile"
```
```suggestion
	turret 56 96 "Warder Anti-Missile"
```

Saugia commented on data/korath/korath ships.txt:
```diff
@@ -387,6 +387,172 @@ ship "'olofez"
+	turret 0 14 "Shunt-Strike Turret"
+	turret -145.5 83 "Shunt-Strike Turret"
+	turret 145.5 83 "Warder Anti-Missile"
+	turret -44.5 113 "Warder Anti-Missile"
```
```suggestion
	turret -146 83.5 "Warder Anti-Missile"
```

Saugia commented on data/korath/korath ships.txt:
```diff
@@ -387,6 +387,172 @@ ship "'olofez"
+	turret -145.5 83 "Shunt-Strike Turret"
+	turret 145.5 83 "Warder Anti-Missile"
+	turret -44.5 113 "Warder Anti-Missile"
+	turret 44.5 113 "Warder Anti-Missile"
```
```suggestion
	turret 146 83.5 "Shunt-Strike Turret"
```

Saugia commented on data/korath/korath ships.txt:
```diff
@@ -387,6 +387,172 @@ ship "'olofez"
+	turret 145.5 83 "Warder Anti-Missile"
+	turret -44.5 113 "Warder Anti-Missile"
+	turret 44.5 113 "Warder Anti-Missile"
+	turret -139.5 178 "Warder Anti-Missile"
```
```suggestion
	turret -141.5 177 "Shunt-Strike Turret"
```

Saugia commented on data/korath/korath ships.txt:
```diff
@@ -387,6 +387,172 @@ ship "'olofez"
+	turret -44.5 113 "Warder Anti-Missile"
+	turret 44.5 113 "Warder Anti-Missile"
+	turret -139.5 178 "Warder Anti-Missile"
+	turret 139.5 178 "Shunt-Strike Turret"
```
```suggestion
	turret 141.5 177 "Warder Anti-Missile"
```

Saugia commented on data/korath/korath ships.txt:
```diff
@@ -387,6 +387,172 @@ ship "'olofez"
+		"Systems Core (Large)"
+		"Large Heat Shunt" 2
+		"Small Heat Shunt" 2
+		"Korath Repeater Rifle"  957
```
```suggestion
		"Thermal Repeater Rifle"  986
```

Zitchas commented:
Expanding on that: it fits my story plans best of it is suited to independent operation. This isn't merely a "I can work with it if it can" but rather "It is problematic to work with it if it isn't" sorry of thing. In other words, I want this to be have statistics suitable to independent operation in hostile territory with minimal to no escorts.

ravenshining commented:
There seemed to be some confusion regarding the Ikatila'ej.  People were assuming that, as in 4987, the cagey thing on the front was a factory that produced microbots, rather than the ship as a whole being a factory that uses microbots as I intend, with the front bit just being the intake.  We left the name because "factory" seemed well enough, but it's been causing confusion.  So, I've renamed it to "Microbot Grinder" and added a small note to the ship desc about its use.

Not mentioned in any of the above comments, now that we have the `merciful` personality, I've marked the rescue fleets as `merciful`.  They've got more important things to do than to go chasing after former hostiles.

warp-core commented on data/korath/korath ships.txt:
```diff
@@ -387,6 +387,164 @@ ship "'olofez"
+		"weapon capacity" 440
+		"engine capacity" 259
+		"spinal mount" 1
+		"capture defense" 800
```
I'm not sure capture defense works here. When doing the calculations, I think it traverses the list of outfits on the ship. I don't think the base attributes of the ship are not included in that.

Hurleveur replied:
Yeah thats why it should be on the outfit
Honestly all syscores should have capture defence.

warp-core commented on data/korath/korath ships.txt:
```diff
@@ -387,6 +387,164 @@ ship "'olofez"
+		"engine capacity" 259
+		"spinal mount" 1
+		"capture defense" 800
+		illegal 40000000
```
Does the code for fining on scan actually check the base attributes of the ship or is it just the outfits?

Saugia replied:
With similar large crew, and me placing 26 Stations on it (more than most I feel), it should be notably hard to capture I think. And since these are rare, I think that should be good

Hurleveur replied:
Just make the gun illegal if we want to I'd say

Hurleveur replied:
Also its hard to capture only if you dont have the other ship
If you do you can get this one without too much trouble, the microbot count could be invcreased by 14 easily

Saugia replied:
I think it's fine as is for now, and we can rely on future QoL outfits (or buff MDS's) to fix it. Derpy doesn't want us to increase the outfit space much further.

ravenshining commented:
Reverted the intrinsic defense and illegality because, as warp-core pointed out, they don't actually work.

I don't want to put it on the weapon, or at least nowhere near that strongly.  We can do that and add some to the upcoming QoL outfits, perhaps even the systems cores, in a future PR.

Saugia reviewed (approved):
My only remaining concern is how I had set up the hardpoint sprite- in order for it to work the way I had it, the gun port had to be in the middle of the mouth. I was under the assumption the factory wasn't going to be removed, so I will likely revisit this in a future PR to tidy that up. 
![hardpoint](https://user-images.githubusercontent.com/93169396/201229097-77fbe633-6c52-4890-b017-b1b7dc068ae7.JPG)

Saugia commented on copyright:
```diff
@@ -1681,9 +1681,13 @@ Comment: Derived from works by Michael Zahniser (under the same license) and Lia
 
 Files:
```
```suggestion
Files:
 images/hardpoint/microbot?factory*
```
